azure-pipelines.yml

```diff
@@ -120,18 +121,15 @@ jobs:
 
   - script: |
       python -m pip install --upgrade pip setuptools
-    failOnStderr: true
     displayName: 'Upgrade pip and setuptools'
 
   - script: |
       pip install -e ".[tests, doc]"
-    failOnStderr: true
     displayName: 'Install dev environment'
 
   - script: |
-      pytest --cov gtda --cov-report xml
-      flake8
-    failOnStderr: true
+      pytest --cov gtda --cov-report xml || exit /b
```

Comment:
@rth could you comment on these additions?

Reply:
This says, if pytest command exits with a non 0 exit code (i.e. it failed), explicitly exit this script with an error (and don't run following steps). It's equivalent to setting `set -e` on Linux, but for Windows.
see https://stackoverflow.com/a/13863374

Reply:
Thanks for the explanation!